src/socket.cpp

```diff
@@ -39,15 +46,15 @@ void CSocket::Init ( const quint16 iPortNumber )
 #endif
 
     // create the UDP socket
-    UdpSocket = socket ( AF_INET, SOCK_DGRAM, 0 );
+    UdpSocket = socket ( AF_INET6, SOCK_DGRAM, 0 );
```

Comment:
The `IPV6_V6ONLY` socket option must be false in order for the socket to listen on both protocols. On Linux it's false by default on most (all?) distros, but on Windows it is true by default, see https://docs.microsoft.com/en-us/windows/win32/winsock/ipproto-ipv6-socket-options.